### DISCUSSION TITLE
Search enhancement

### DISCUSSION THREAD

**Discussion by KadenT8** [wstyres/Zebra#192] [Ideas]
I got the modmyi through community repos and when searching zeppelin it comes up with a bunch of packs but not the actual tweak. Is there a way to refine the search to just what Im looking for?

**Comment by KadenT8** [comment]
One idea to fix this is you could add “quotations” around it and then it will only search for that specific phrase

**Comment by worldlydev** [comment]
I agree that search filters should be added. I use them all the time in Google Search and feel that they are underutilized by many. This would go a long way towards making Zebra my preferred package manager which is something I want to happen due to the fact that Cydia feels clunky.

**Comment by worldlydev** [comment]
I was thinking something along the lines of this:
![ZebraSearchFilter](https://user-images.githubusercontent.com/51668635/59616341-85bb3700-9124-11e9-8cbd-05049e554106.jpg)

The ability to filter your search to Type (theme, font, tweak, etc), Source (BigBoss, Packix, etc), Developer (Nepeta, Spark,etc) and Time (last update) is in my opinion the ultimate tool for a power user such as me.

Adding small features like this would definitely make Zebra my favorite package manager. Not to say that it isn't already 😉
